src/main/java/com/google/api/codegen/transformer/FeatureConfig.java

```diff
@@ -38,7 +38,7 @@ public boolean enableMixins() {
 
   /** Returns true if streaming APIs are supported. */
   public boolean enableGrpcStreaming() {
-    return false;
+    return true;
```

Comment:
Yeah, my concern is that when we add another language down the line, it won't have gRPC streaming support and we'll have to put this method back in.